Documentation/articles/getting_started/1_setting_up_your_development_environment_ubuntu.md

```diff
@@ -0,0 +1,87 @@
+# Setting up your development environment for Ubuntu 20.04
+
+In this section we will go over setting up your development environment for Ubuntu 20.04.
+
+## Install .NET Core SDK:
+
+Add repository:
+```sh
+wget https://packages.microsoft.com/config/ubuntu/20.04/packages-microsoft-prod.deb -O /tmp/packages-microsoft-prod.deb
+sudo dpkg -i /tmp/packages-microsoft-prod.deb
+sudo apt update
+```
+
+Install packages:
+```sh
+sudo apt-get install -y apt-transport-https
+sudo apt-get install -y dotnet-sdk-3.1
+```
+
+## [Optional] Install mono:
+
+Mono is a C# runtime, just like .NET Core, and it is not nessesary to install it if you are just targeting Linux, however if you want to target some other platforms, like Android, it will be required.
```

Comment:
Is this necessary? It seems to me this isn't very valuable until we have a full guide to set up Android development on Linux since it's not very straightforward and officially not supported by Xamarin.

Reply:
Its actually straightforwards to set it up on Ubuntu 20.04, they provide a .deb package that you just install.

Reply:
Xamarin.Android you mean?

Reply:
Yes.